src/ethereum_test_fixtures/blockchain.py

```diff
@@ -401,13 +403,29 @@ def from_withdrawal(cls, w: WithdrawalGeneric) -> "FixtureWithdrawal":
         return cls(**w.model_dump())
 
 
+@dataclass(slots=True)
+class WitnessChunk:
+    """Represents execution witness data for a block."""
+
+    state: List[str]
+    codes: List[str]
+    keys: List[str]
+    headers: List[str]
+
+    @classmethod
+    def from_json(cls, s: str) -> List["WitnessChunk"]:
+        """Parse witness chunks from JSON string."""
+        return [cls(**obj) for obj in json.loads(s)]
```

Comment:
This function is a bit misleading since it returns a list of instances of the object instead of single instance.